### PR TITLE
feature: add markdownlint as a supported tool CY-4001

### PR DESCRIPTION
**Pull request by pedrocodacy**
Added as supported tool in
	- getting-started/supported-languages-and-tools
	- related-tools/codacy-plugin-tools
Added as tool with fix suggestions enabled in repositories-configure/github-integration#suggest-fixes

- [ ] wait for CY-3953 to be finished before merging

### CODE REVIEW

**Review by pedrocodacy — COMMENTED** [review]
Wait for CY-3953 to be finished before merging

File: docs/getting-started/supported-languages-and-tools.md
Comment (prcr) [inline docs/getting-started/supported-languages-and-tools.md:195]: The new tool is missing in the column "Static analysis":

```suggestion
      <td><a href="https://github.com/remarkjs/remark-lint">RemarkLint</a>, <a href="https://github.com/markdownlint/markdownlint">markdownlint</a></td>
```

File: docs/getting-started/supported-languages-and-tools.md
Comment (prcr) [inline docs/getting-started/supported-languages-and-tools.md:196]: Add the missing "wrench icon" with a link to an explanation of the suggested fixes feature:

image

```suggestion
      <td><a href="https://github.com/markdownlint/markdownlint">markdownlint</a>   <a href="#suggested-fixes">🔧</a></td>
```

File: docs/related-tools/codacy-plugin-tools.md
Comment (prcr) [inline docs/related-tools/codacy-plugin-tools.md:147]: The name of the tools and our corresponding repositories are mixed up:

image

```suggestion
<td><a href="https://github.com/markdownlint/markdownlint">markdownlint</a></td>
<td><a href="https://github.com/codacy/codacy-markdownlint" class="skip-vale">codacy/codacy-markdownlint</a></td>
</tr>
<tr>
<td><a href="https://github.com/remarkjs/remark-lint">RemarkLint</a></td>
```

File: docs/repositories-configure/integrations/github-integration.md
Comment (prcr) [inline docs/repositories-configure/integrations/github-integration.md:72]: Fix small typo:

```suggestion
    -   The only two tools that suggest fixes are [ESLint](https://eslint.org/docs/rules/) and [MarkdownLint](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md). However, we're planning to support suggestions from more tools.
```

**Review by prcr — CHANGES_REQUESTED** [review]
The maintainers of the tool aren't very consistent with the way they capitalize the name of the tool, but [most of the time they use all lowercase](https://github.com/markdownlint/markdownlint), so I would consider the official name of the tool to be **markdownlint**.

We also need to add the name of the tool to the file `.github/styles/Vocab/Codacy/accept.txt` so that the spellchecking step of the CI doesn't fail. :nerd_face:

**Review by prcr — APPROVED** [review]

**Review by prcr — DISMISSED** [review]

**Review by franciscodua — APPROVED** [review]

File: docs/repositories-configure/integrations/github-integration.md
Comment (prcr) [inline docs/repositories-configure/integrations/github-integration.md:72]: Correct capitalization of markdownlint and fix link for rules reference:

```suggestion
    -   The only two tools that suggest fixes are [ESLint](https://eslint.org/docs/rules/) and [markdownlint](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md). However, we're planning to support suggestions from more tools.
```

File: docs/getting-started/supported-languages-and-tools.md
Comment (prcr) [inline docs/getting-started/supported-languages-and-tools.md:195]: The tool is missing under the column **Static Analysis**:

```suggestion
      <td><a href="https://github.com/remarkjs/remark-lint">RemarkLint</a>, <a href="https://github.com/DavidAnson/markdownlint">markdownlint</a></td>
```

**Review by prcr — CHANGES_REQUESTED** [review]

**Comment by franciscodua** [comment]
@prcr I just accepted your suggestions. Ready for another review 😄

**Review by prcr — APPROVED** [review]
:shipit: :tada:

**Pull request merged**
